src/main/java/view/UserPageImpl.java

```diff
@@ -1,35 +1,49 @@
 package view;
 
 import model.BlindHorizontal;
+import model.BlindVertical;
+import service.DataWriter;
 import service.HorizontalService;
-import service.HorizontalServiceImpl;
-import service.PriceCatalog;
+import service.VerticalService;
 
+import java.io.File;
 import java.util.Scanner;
 
 public class UserPageImpl implements UserPage {
 
-    private PriceCatalog priceCatalogImpl;
     private HorizontalService horizontalServiceImpl;
+    private VerticalService verticalServiceImpl;
+    private DataWriter dataWriter = new DataWriter();
 
-    public UserPageImpl(HorizontalService horizontalServiceImpl, PriceCatalog priceCatalogImpl) {
+    public UserPageImpl(HorizontalService horizontalServiceImpl, VerticalService verticalServiceImpl) {
         this.horizontalServiceImpl = horizontalServiceImpl;
-        this.priceCatalogImpl = priceCatalogImpl;
+        this.verticalServiceImpl = verticalServiceImpl;
     }
 
     Scanner scanner = new Scanner(System.in);
     Scanner scannerStr = new Scanner(System.in);
 
     @Override
     public void showBlindCost() {
-        priceCatalogImpl.initDate();
+        File file = new File(".\\src\\main\\resources\\output.csv");
```

Comment:
please, add all URIs to the application.properties

Reply:
all URL added to application.properties